android/libraries/rib-android-core/src/main/kotlin/com/uber/rib/core/CoreAppCompatActivity.kt

```diff
@@ -81,4 +85,16 @@ abstract class CoreAppCompatActivity : AppCompatActivity() {
     super.onActivityResult(requestCode, resultCode, data)
     activityDelegate?.onActivityResult(this, requestCode, resultCode, data)
   }
+
+  /**
+   * [RibActivity] must call this since it does not use [ComponentActivity.setContentView] which
```

Comment:
What happens for activities that extend from this and call setContentView? I imagine lifecycle owners get overwritten and that's ok

Reply:
overwritten & ok